src/styles/blocks/recommended.scss

```diff
@@ -0,0 +1,70 @@
+.recommended {
+  margin-bottom: 120px;
+
+  @include onTablet {
+    margin-bottom: 128px;
+  }
+
+  &__title {
+    margin-bottom: 48px;
+
+    @include title;
+
+    @include onTablet {
+      margin-bottom: 64px;
+    }
+  }
+
+  &__content {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+
+    @include onDesktop {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      column-gap: 24px;
+    }
+  }
+
+  &__card:not(:last-child) {
+    margin-bottom: 48px;
+
+    @include onTablet {
+      margin-bottom: 40px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 0;
+    }
+  }
+
+  &__image {
+    margin-bottom: 32px;
+    width: 324px;
+    height: 320px;
+  }
+
+  &__subtitle {
+    margin: 0;
+    padding-bottom: 4px;
+    font-weight: 700;
+    border-bottom: 1px solid transparent;
+
+    &:hover {
```

Comment:
You can avoid using 'display: inline-block;' here as it doesn't seem to be necessary. The border color change will still work without it. ✨